CRM/Core/Invoke.php

```diff
@@ -331,13 +331,17 @@ public static function runItem($item) {
   /**
    * This function contains the default action.
    *
+   * Unused function.
+   *
    * @param $action
    *
    * @param $contact_type
    * @param $contact_sub_type
    *
+   * @Deprecated
    */
   public static function form($action, $contact_type, $contact_sub_type) {
+    CRM_Core_Error::deprecatedWarning('unused');
```

Comment:
Do I assume correctly that this error message -- "unused" -- is a temporary placeholder ("Let's see if any tests fail with this warning...")... and that it will be replaced with a real error message?

Reply:
@totten it means there is no alternative function - no-one should ever hit this message because the function is never called - but it just hangs around briefly on the off chance some extension is doing something weird. I'm about 95% we could just remove this so any form of deprecation error is cautious